Documentation/Game Design.txt

```diff
@@ -16,43 +16,77 @@ UI:
   => Map
     - Cursor selects planets
     - Arrow keys control ship
+
   => Current planet info
-    - Displays info when a ship collides with a planet on the map
+    - Displays planet info when a ship collides with a planet on the map
+
   => Selected planet info
-    - Displays info when a player hovers the cursor over a planet
+    - Displays planet info when a player hovers the cursor over a planet
+
   => Dialogue box
     - Displays dialog to player during events
 
 Gameplay:
   => Exploration
-    - Move ship around the map to visit each planet
-    - Movement on the map consumes fuel
+    - Planets
+      > Galaxy contains fixed number of planets
+      > Planets can be undiscovered, discovered, colonized (state machine)
+        + Undiscovered planets can't be selected
+        + Discovered planets become colonized if population is > 0, vice versa
+    - Ship
+      > Ship has fixed population
+      > Movement on the map consumes fuel
+      > Move ship around the map to visit each planet
+      > Ship population temporarily added to planet when landed
+      > Ship must be on planet to receive minerals for fuel
 
   => Resource Management
-    - Planet Resources:
-      > Food
-      > Deposits
+    - Population
+      > Planet max population limited by infrastructure, uniformly set on discovery
+      > Planet birth and death rate dependant on excess food, slightly random
+      > People consume fixed amount of food per frame
+      > People produce fixed amount of food per frame
+      > Player chooses ratio of farmers to miners per planet
+      > Player chooses ratio of minerals (mined deposits) used for infrastructure
+    - Food
+      > Planet has fixed max food production per frame, randomly set
+    - Deposits
+      > Planet has fixed total deposits that can be mined, randomly set
+      > Miners turn deposits into usable minerals, which can be used as fuel
+      > Minerals can be used to build infrastructure and raise max population limit
 
-    - Ship Resources:
-      > Population
-      > Fuel
-
-    - Ship must be on planet to receive a planet's resources
-    - Deposits can be used to create fuel for the ship
-
-  => Population Growth
-    - Grows and shrinks in real time through births and deaths
-    - Food must sustain population
+  => Random Events
+    - Random events affect food, mining, births, deaths
 
-  => Planets
-    - Each planet has a specific amount of each resource
-    - Can be colonized by sending population to surface
-      > Population can be used to gather resources
-    - Events/disasters can happen periodically
+  => Alien Warship
+    - Alien warship roams and can potentially wipe planet population
 
   => Endgame:
-    - Discover all planets
+    - Maximizing population gives score
+    - Discover all planets, "win condition" bonus?
```

Comment:
I mean, if they're gonna discover all the planets, they're already going to have a higher score than if they didn't. So why need a bonus? Unless it's used to entice them not to lose

Reply:
Well discovering planets doesn't add to their score except by enabling a higher population, which they may or may not take the time to do. The idea is yes, to make them want to discover all the planets to "win". This adds to the challenge and gives motivation to discover every planet